modules/shortlinks.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * Module Name: WP.me Shortlinks
- * Module Description: Generates shorter links so you can have more space to write on social media sites.
+ * Module Description: Generates shorter links using the wp.me domain.
```

Comment:
Noting that the translation for this get auto-generated by a build script by the looks of it - let me know if I need to run that as part of the PR.

Reply:
> let me know if I need to run that as part of the PR.

You'll indeed need to commit the changes to module-headings.php that this PR causes.

Thanks!